tests/conda_token/test_token_cli.py

```diff
@@ -13,6 +13,8 @@
 
 # ruff: noqa: E402
 from anaconda_auth._conda import repo_config
+from anaconda_auth._conda.repo_config import token_list
+from anaconda_auth._conda.repo_config import token_set
```

Comment:
We can consolidate to just use `repo_config.token_list` etc. to minimize number of import statements.